// @unocss/preset-mini/dist/variants.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

const _default = require('./shared/preset-mini.52ab4b0f.cjs');
require('./shared/preset-mini.f9ff00ee.cjs');
require('@unocss/core');
require('./shared/preset-mini.7086e6db.cjs');



exports.calcMaxWidthBySize = _default.calcMaxWidthBySize;
exports.partClasses = _default.partClasses;
exports.variantAria = _default.variantAria;
exports.variantBreakpoints = _default.variantBreakpoints;
exports.variantColorsMediaOrClass = _default.variantColorsMediaOrClass;
exports.variantCombinators = _default.variantCombinators;
exports.variantContainerQuery = _default.variantContainerQuery;
exports.variantCssLayer = _default.variantCssLayer;
exports.variantCustomMedia = _default.variantCustomMedia;
exports.variantDataAttribute = _default.variantDataAttribute;
exports.variantImportant = _default.variantImportant;
exports.variantInternalLayer = _default.variantInternalLayer;
exports.variantLanguageDirections = _default.variantLanguageDirections;
exports.variantNegative = _default.variantNegative;
exports.variantPrint = _default.variantPrint;
exports.variantPseudoClassFunctions = _default.variantPseudoClassFunctions;
exports.variantPseudoClassesAndElements = _default.variantPseudoClassesAndElements;
exports.variantScope = _default.variantScope;
exports.variantSelector = _default.variantSelector;
exports.variantSupports = _default.variantSupports;
exports.variantTaggedPseudoClasses = _default.variantTaggedPseudoClasses;
exports.variantVariables = _default.variantVariables;
exports.variants = _default.variants;
